### PR TITLE
Modularize dcli.py and Fix a bug in view-files command

### PR DESCRIPTION
**Pull request by raghav-dalmia**
**Modularize dcli.py**
-----
I have created two modules [auth.py](https://github.com/raghav-dalmia/drive-cli/blob/master/drive_cli/auth.py) and [actions.py](https://github.com/raghav-dalmia/drive-cli/blob/master/drive_cli/actions.py). 

`auth.py` contains all the functions related to authentication of eamil i.e. `login()` and `logout()` function. `actions.py` contains all the functions related to subcommands like `view-files`, `ls`, `clone` and others.

Also, I put all the other helper functions in `utils.py` and import them into `actions.py`

**Fix a bug in view-file command**
-----
Following code is written twice in the view-file function due to which there is modification the query. I have deleted one of them.

        if pid:
            parent = click.prompt('enter the fid of parent or  sharing link')
            fid = utils.get_fid(parent)
            if (name != False) or (types != False):
            query += " and "
            query += "'" + fid + "' in parents"`

@nurdtechie98 @inishchith please review the changes.

### CODE REVIEW

**Comment by raghav-dalmia** [comment]
@nurdtechie98 I didn't get your point. The proper message should be, with each commit or with PR?

**Comment by nurdtechie98** [comment]
> @nurdtechie98 I didn't get your point. The proper message should be, with each commit or with PR?

Pull request, Always write a short description of what the pull request does, link any issues that is solving in this case #10 so that it becomes easy to understand and review

**Comment by inishchith** [comment]
@raghav-dalmia Thanks for your PR. 

Apart from what @nurdtechie98 suggested.
I'd really appreciate if you could follow the following convention.

1. No `subcommand` directory, please place everything under `drive_cli`.
2. It'd be better if we structure the files something like - 
```
           auth.py
           | - login()
           | - logout()
           actions.py
           | - push() 
           | - pull()
           utils.py
           | - mimetypes
           | - other helpers ...
```
 -  Limiting them to no more than 3-5 files, i hope you get the gist.
3. All `file names` must be lowercase .

Really appreciate your efforts, i hope you don't mind making the above changes.

Let me know what you think ;)

**Comment by nurdtechie98** [comment]
> @raghav-dalmia Thanks for your PR.
> 
> 2. It'd be better if we structure the files something like -
> 
> ```
>            auth.py
>            | - login()
>            | - logout()
>            actions.py
>            | - push() 
>            | - pull()
>            utils.py
>            | - mimetypes
>            | - other helpers ...
> ```
This looks better

**Comment by raghav-dalmia** [comment]
@nurdtechie98 @inishchith agree with your point. File Structure your post here also looks good to me. I will make changes accordingly.

**Comment by nurdtechie98** [comment]
@raghav-dalmia good job, LGTM.
Only thing to be noted next time make sure squash all your commits. It's a generally a good practice to follow.

**Comment by raghav-dalmia** [comment]
I have resolved all the conflicts, please review it. @nurdtechie98

**Comment by nurdtechie98** [comment]
Awesome 🎉  LGTM

File: drive_cli/auth.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import click
+import pyfiglet
+import requests
+from oauth2client import file, client, tools
+
+SCOPES = 'https://www.googleapis.com/auth/drive'
+dirpath = os.path.dirname(os.path.realpath(__file__))
+
+
+def LOGIN(remote):
```
Comment (inishchith) [inline drive_cli/auth.py]: Please lowercase the method name.

File: drive_cli/actions.py [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+import click
+from pick import Picker
+from httplib2 import Http
+from oauth2client import file
+from prettytable import PrettyTable
+from googleapiclient.discovery import build
+
+dirpath = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dirpath)
+import utils
```
Comment (inishchith) [inline drive_cli/actions.py]: push this line on the top [ import section ]

Comment (raghav-dalmia) [reply drive_cli/actions.py]: can't push these line on the top, as line 10,11 are using functions of os and sys library.

File: drive_cli/auth.py (hunk 1 above)
Comment (raghav-dalmia) [reply drive_cli/auth.py]: Done

File: drive_cli/actions.py (hunk 2 above)
Comment (nurdtechie98) [reply drive_cli/actions.py]: @raghav-dalmia I guess @inishchith is suggesting moving just line 12 up

Comment (raghav-dalmia) [reply drive_cli/actions.py]: ok, got it

Comment (raghav-dalmia) [reply drive_cli/actions.py]: done

**Pull request merged**